tests/utils.ts

```diff
@@ -20,9 +25,24 @@ export function generateVarName(prefix = 'v'): string {
  * @param tableName Name of the variable to assign the table to
  * @returns String of the command to create a table
  */
-export function makeTableCommand(tableName = generateVarName('t')): string {
-  return `from deephaven import empty_table
+export function makeTableCommand(
+  tableName = generateVarName('t'),
+  type = TableTypes.Number
+): string {
+  switch (type) { 
+    case TableTypes.StringAndNumber:
+      return `from deephaven import new_table
+from deephaven.column import string_col, double_col
+
+${tableName} = new_table([
+double_col("Doubles", [3.1, 5.45, -1.0, 1.0, 3.0, 4.20]),
```

Comment:
formatting looks odd on the string but if I have any indents it causes errors when it is entering the command into the console. Not sure how I could improve readability here or if its necessary. I could make it all single line to avoid any indents but that would create super long strings